### ISSUE TITLE
Pull 2D or 3D info from chosen app's config.json

### ISSUE THREAD

**Issue by gavinr** [Esri/generator-esri-appbuilder-js#55]
After #54, the app asks if the app is 2D or 3D. If we moved that prompt to _after_ asking for the WABROOT and app, then we could know if it's 2D or 3D based on the app's `config.json`. We would still need to prompt the user for 2D or 3D in the case that "none" is selected on the "Web AppBuilder Application" prompt.

**Comment by tomwayson** [comment]
What config prop would you use, `map.2D` and `map.3D`? That's the only thing I can find that's consistent through the versions.

In #54 2d/3d choice is used to set default WAB install root, which comes before app choice. However, I agree that it would be more useful to have it the other way around, b/c we're more likely to be right about 2d/3d than WAB install root.

I would say we should probably do this, but still _always_ show the 2d/3d prompt, just get the default from the app (if any). 

The question is when to do it. We're one click of the merge button away from being done w/ 2.0. I'm more interested in [ES6](https://github.com/Esri/generator-esri-appbuilder-js/issues/32) than this. However, I don't know how much it would matter to people if we moved their cheese a little later down the road (2.2) etc.

**Comment by gavinr** [comment]
I don't see it as a high priority ... more of a "nice to have"

**Comment by gavinr** [comment]
As this is super minor, and the whole "specifying of 2d-3d thing" may change down the road if the JSAPI 4.x is used for everything, I think we should just close this for now. Feel free to comment if anyone disagrees.
